.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 130
```

Comment:
We'd usually stick to 120. If this makes it a nightmare then this is still better than nothing.

Reply:
There were a couple lines that were ~125 so thought I'd bring the fudge